### PR TITLE
Translate - French

### PR DESCRIPTION
**Pull request by Alfred-Neuman**
**When merged this pull request will:**
- Describe what this pull request will do
- Each change in a separate line
- Respect the [Submitting Content Guidelines](https://github.com/CBATeam/CBA_A3/wiki/Submitting%20content)

### CODE REVIEW

File: addons/strings/stringtable.xml [hunk 1]
```diff
@@ -34,12 +34,8 @@
         </Key>
         <Key ID="STR_CBA_FORMAT_NUMBER_THOUSANDS_SEPARATOR">
             <English>,</English>
-            <Czech> </Czech>
             <German>.</German>
-            <Polish> </Polish>
-            <Russian> </Russian>
-            <French> </French>
-            <Spanish> </Spanish>
+            <French>,</French>
```
Comment (jonpas) [inline addons/strings/stringtable.xml:38]: Don't delete other languages. Separator is a space, that is fine.

**Comment by Alfred-Neuman** [comment]
I think it's okay.

File: addons/strings/stringtable.xml [hunk 2]
```diff
@@ -38,7 +38,7 @@
             <German>.</German>
             <Polish> </Polish>
             <Russian> </Russian>
-            <French> </French>
+            <French>,</French>
```
Comment (PabstMirror) [inline addons/strings/stringtable.xml:41]: Are you sure about this, it doesn't seem to match some sources
https://en.wikipedia.org/wiki/Decimal_separator

Comment (Alfred-Neuman) [reply addons/strings/stringtable.xml]: I confused it with the decimal separator

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**